static/js/around-map.js

```diff
@@ -0,0 +1,46 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoibmVvbWFkIiwiYSI6ImNqMHRrZ3ZwdzAwNDgzMm1kcHRhMDdsZGIifQ.bOSlLkmc1LBv0xAbcZXpog';
+const map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/mapbox/streets-v9',
+  center: [-10, 45],
+  zoom: 2
+})
+
+// users interesting points
+const giveData = new Worker("/static/js/webworker-around.js");
+
+giveData.onmessage = (informations) => {
+  const [name,wifi,power,comment,lng,lat] = informations.data
+
+  const el = document.createElement('div')
+  el.classList.add('marker')
+
+  let content = "<h2>"+name+"</h2><p>Wifi quality: "+ wifi+"<br>Power available: "+power+"<br>Comments: "+comment+"</p>"
```

Comment:
You're using ES6, therefore you should use [template strings](https://babeljs.io/learn-es2015/#ecmascript-2015-features-template-strings)